backend/app.ts

```diff
@@ -12,7 +14,7 @@ import { MatchReporter } from './match-reporter/MatchReporter'
 const jsonParser = bodyParser.json()
 const urlencodedParser = bodyParser.urlencoded({ extended: false })
 
-const app = express()
+const app: Application = express()
 const port = 3000
```

Comment:
just as as aside - it would be good to stick it into an env in the future

Reply:
filed https://github.com/salsita/foosball-rating/issues/127